plugins/modules/notification/cisco_webex.py

```diff
@@ -111,9 +116,9 @@
 from ansible.module_utils.urls import fetch_url
 
 
-def spark_message(module):
```

Comment:
Is there a reason why you rename this function to `spark_msg`? Either keep `spark_message`, or rename it to `webex_message`.